scripts/complete_liquidity_provision.js

```diff
@@ -84,6 +92,8 @@ module.exports = async (callback) => {
     const { instance: targetToken, decimals: targetTokenDecimals } = targetTokenData
     const { instance: stableToken, decimals: stableTokenDecimals } = stableTokenData
 
+    const dry_run = argv["dry-run"]
```

Comment:
Any particular reason you chose to access like a dict instead of like a JS object?

Reply:
I actually made it return the fleet address on top of emitting them as an event. This way we can "call" the method in the dry run mode.

Reply:
multi word command line arguments are by convention kebab-case and that cannot be directly accessed on the argv as far as I know.

Reply:
Interesting. Does this mean we should consider changing our other "multi-word" arguments from camel case?